### PR TITLE
Update skip-building-some-commits-with-ci-skip.md

### PR DESCRIPTION
**Pull request by dradulescu**

### CODE REVIEW

**Review by AleksandarCole — APPROVED** [review]
Looks good.

File: docs/essentials/skip-building-some-commits-with-ci-skip.md [hunk 1]
```diff
@@ -7,3 +7,9 @@ description: If you'd like your commit, or a series of commits that you're pushi
 If you'd like your commit, or a series of commits that you're pushing, to not
 trigger a build, just write `[ci skip]` or `[skip ci]` somewhere in your 
 commit's message.
+
+!!! info "Usage with squashed commits"
+	When squashing commits before merging in another branch all commit messages
+will be merged as well. If a previous commit in the current branch contains
+`[ci skip]` or `[skip ci]` it will be propagated and the merged commit wil
```
Comment (radwo) [inline docs/essentials/skip-building-some-commits-with-ci-skip.md:14]: `wil` -> `will`

Comment (dradulescu) [reply docs/essentials/skip-building-some-commits-with-ci-skip.md:14]: Thanks @radwo there's also a problem with the formatting.

**Comment by dradulescu** [comment]
I can't edit the files in this branch anymore since it's been merged so opened this new PR: https://github.com/semaphoreci/docs/pull/933

**Pull request merged**